imap/jmap_mail.c

```diff
@@ -124,83 +124,37 @@ static int jmap_emailheader_getblob(jmap_req_t *req, jmap_getblob_context_t *ctx
  */
 
 static jmap_method_t jmap_mail_methods_standard[] = {
+    {"Email/query",        JMAP_URN_MAIL, &jmap_email_query,        JMAP_NEED_CSTATE},
```

Comment:
I guess it's no surprise that I am much in favor of using automated formatting, but the exact format config I am not much picky about. But these lines are a good example of where I see clang-formatting lacking, probably because of some missing config: I much prefer multiple entries in a struct or array literal to be formatted equally, e.g. if one of them doesn't fit on a line, then all other entries should follow that format, too. 
We could either find a config that does this for us, or I would suggest we guard these declarations with `// clang-format on/off` comments.

Reply:
Sorry, I meant to comment on all the lines in the declaration of `jmap_mail_methods_standard` but somehow that didn't work.